### DISCUSSION TITLE
Event for autorotate

### DISCUSSION THREAD

**Discussion by annahassel** [google/model-viewer#4162] [Ideas]
Hello! Is it possible to add event for auto rotate, like `camera-change` event? I need to change rotation speed or direction depending on `turntableRotation`. I tried to use `setInterval` or `requestAnimationFrame` but it does'n work in inactive tab. Or may be there is another way to handle rotation angle?

**Comment by elalish** [comment]
Yes, a `turntable-rotation` event sounds quite reasonable. Would you like to make a PR to add it?

**Reply by annahassel** [reply]
Yes, I can do it. Thank you!
